src/fsharp/FSharp.Build/FSharp.Build.fsproj

```diff
@@ -35,16 +35,24 @@
     <NoneSubstituteText Include="Microsoft.FSharp.NetSdk.props" CopyToOutputDirectory="PreserveNewest">
       <TargetFileName>Microsoft.FSharp.NetSdk.props</TargetFileName>
       <Pattern1>{{FSharpCoreShippedPackageVersion}}</Pattern1>
-      <Replacement1>$(FSharpCoreShippedPackageVersion)</Replacement1>
+      <Replacement1>$(FSharpCoreShippedPackageVersionProperty)</Replacement1>
       <Pattern2>{{FSharpCorePreviewPackageVersion}}</Pattern2>
-      <Replacement2>$(FSharpCorePreviewPackageVersion)</Replacement2>
-      <Pattern3>{{FSCorePackageVersion}}</Pattern3>
+		<Replacement2>$(FSharpCorePreviewPackageVersion)</Replacement2>
+		<Pattern3>{{FSCorePackageVersion}}</Pattern3>
```

Comment:
nit: should've have been changed, right?

Reply:
yep , shouldn't have changed. Good eyes.